include/tvm/script/printer.h

```diff
@@ -0,0 +1,54 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+#ifndef TVM_SCRIPT_PRINTER_H_
+#define TVM_SCRIPT_PRINTER_H_
+
+#include <tvm/node/node.h>
+#include <tvm/node/object_path.h>
+
+namespace tvm {
+namespace script {
```

Comment:
Just curious, what do you think works better, `tvm::script::printer` or `tvm::script`

Reply:
If the function name is `Script`, I think `tvm::script::printer` is a better choice. Otherwise the purpose of this function is less clear, because it's a free function rather than a method on IR node.